docs/user-manuals/network/edge-pod-network.md

```diff
@@ -2,3 +2,96 @@
 title: Edge pod network
 ---
 
+## 背景说明
+
+在边缘场景中，为了确保云边网络断开时节点上的业务仍能稳定运行，OpenYurt提供了边缘节点自治功能。该功能能够实现云边断网情况下，业务Pod的异常自动重启、节点重启自动拉起业务Pod等功能。然而，为了保证云边断网后业务Pod和节点的重启不影响边边的容器网络，这里有一些网络情况需要适配。
+
+## Flannel： VTEP MAC地址保持
+### 场景
+如果我们使用flannel作为CNI插件，且后端为VXLAN模式。在节点上会创建一个VTEP设备（通常命名为": flannel.1），同时VNI和VTEP的信息会被记录到节点的annotations中，供其它节点创建相应的路由和转发规则。
+
+<div align="center">
```

Comment:
Please modify the image format.
like: `![system-architecture](../../../../static/img/docs/core-concepts/prometheus.png) `